packages/media-query/src/media-query.hook.tsx

```diff
@@ -148,3 +150,82 @@ const createMediaQueries = (breakpoints: Dict) => {
     return mediaQuery
   })
 }
+
+type BreakpointValues<T> = { [breakpoint: string]: T }
```

Comment:
Should we include the responsive array style as well?

Reply:
We're enforcing breakpoints in the theme to be an object in v1. It makes the logic a lot easier.